server/src/test/java/keywhiz/service/resources/automation/v2/SecretResourceConfigNewSecretOwnershipStrategyInferTest.java

```diff
@@ -0,0 +1,94 @@
+package keywhiz.service.resources.automation.v2;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.dropwizard.jackson.Jackson;
+import java.io.IOException;
+import java.util.Base64;
+import java.util.UUID;
+import keywhiz.IntegrationTestRule;
+import keywhiz.KeywhizService;
+import keywhiz.TestClients;
+import keywhiz.api.automation.v2.CreateGroupRequestV2;
+import keywhiz.api.automation.v2.CreateSecretRequestV2;
+import keywhiz.api.automation.v2.SecretDetailResponseV2;
+import okhttp3.OkHttpClient;
+import okhttp3.Response;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.rules.RuleChain;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SecretResourceConfigNewSecretOwnershipStrategyInferTest {
```

Comment:
Currently, this test file as well as `SecretResourceConfigNewSecretOwnershipStrategyNoneTest.java` only test `createSecret` function. When `createOrUpdate` function is also modified to infer secret owner, these two test files will include those tests as well. Open to naming suggestions for these test files.

Reply:
As much as I like long, descriptive names for things, that is certainly a mouthful. Maybe SecretResourceInferOwnerTest and SecretResourceNoOwnerTest?